src/librustdoc/clean/inline.rs

```diff
@@ -124,7 +124,7 @@ crate fn try_inline(
     let attrs = merge_attrs(cx, Some(parent_module), target_attrs, attrs_clone);
 
     cx.renderinfo.borrow_mut().inlined.insert(did);
-    let what_rustc_thinks = clean::Item::from_def_id_and_parts(did, Some(name), kind, cx);
+    let what_rustc_thinks = clean::Item::from_def_id_and_parts(did, Some(name.clean(cx)), kind, cx);
```

Comment:
I haven't look deep but I wonder why do we need to have this `.clean(cx)` so many times unlike before?

Reply:
Does the explanation in the commit message help? https://github.com/rust-lang/rust/pull/79372/commits/bbded0bf2a59d6d0f53f8f5c2d0b0d944ef38256

Reply:
Yes, now I understand. I didn't know it is related to interning.